app/controllers/SessionsController.scala

```diff
@@ -62,6 +70,15 @@ class SessionsController @Inject()(messagesApi: MessagesApi,
                                   ) extends KnolxAbstractController(controllerComponents) with I18nSupport {
 
   val usersRepo: UsersRepository = usersRepository

```

Comment:
Please remove this 
val usersRepo: UsersRepository = usersRepository
as it is not required anymore.